### ISSUE TITLE
Feature request: Sorting and Filtering on Explore View

### ISSUE THREAD

**Issue by alfg** [alfg/opendrinks#192]
Now that we have support for keywords, I would like to update our Explore view to support sorting and filtering options.

This will allow the user to to sort the recipe list by name, or optionally filtering down the recipe list by adding keywords or ingredients.

The UI and implementation is up for discussion, but should take advantage of Bootstrap Vue where possible.

Feel free to add any feedback or ideas.

**Comment by TobyAsE** [comment]
I'd like to work on it once #134 is finished.

**Comment by TobyAsE** [comment]
Isn't this a bit redundant with #214 ?
Let's see if I can come up with something that is reasonable and useful regardless.

**Comment by alfg** [comment]
The keywords feature is great, but I think it would still be useful to have filtering controls on the Explore page.

**Comment by TobyAsE** [comment]
I'll build something.
We should maybe only allow filtering for keywords that are used by 2 (whatever number fits) or more drinks, so we don't clutter the filtering-options.

**Comment by TobyAsE** [comment]
See #229

**Comment by TobyAsE** [comment]
As #229 is merged, I believe this issue can be closed.
Could you open a new issue for filtering by multiple keywords?

**Comment by alfg** [comment]
Will do!

**Comment by alfg** [comment]
#248
